specificatie/zakelijk-gerechtigden.yaml

```diff
@@ -141,6 +148,10 @@ components:
           type: "array"
           items:
             $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.2.0/api-specificatie/common.yaml#/components/schemas/HalLink"
+        zakelijkGerechtigdenUitSplitsing:
```

Comment:
Waar is dit veld voor nodig? Je wilt hier toch niet linken naar andere zakelijkGerechtigden?

Reply:
Bij een zakelijkgerechtigde wil je toch de link leggen naar de "bijbehorende" zakelijkgerechtigden (als gevolg van de splitsing van het zakelijk recht dat je hier hebt opgevraagd) Dat is deze link.

Reply:
Volgens mij zou deze link op hetzelfde niveau moeten komen als de self link in ZaakGerechtigdeHalCollectie